app/src/main/kotlin/maru/app/MaruAppFactory.kt

```diff
@@ -300,4 +311,22 @@ class MaruAppFactory {
   private fun ensureDirectoryExists(path: Path) {
     if (!path.exists()) Files.createDirectories(path)
   }
+
+  private fun dbInitialization(
+    beaconGenesisConfig: ForksSchedule,
+    beaconChain: BeaconChain,
+  ) {
+    val qbftForkConfig = beaconGenesisConfig.getForkByConfigType(QbftConsensusConfig::class)
```

Comment:
With this change we will assume that we always have Qbft in the config? Seems like a reasonable assumption. I think we should validating the config somewhere early to ensure that QBFT is configured. Not important for this PR though.